src/Moat/Types.hs

```diff
@@ -418,6 +419,13 @@ data Options = Options
   --
   -- This can be used with @omitFields = const Discard@ to ensure fields are
   -- retained for client compatibility.
+  , deprecatedFields :: [(String, Maybe String)]
```

Comment:
Maybe not necessary to have the comment as part of this since we could comment on the type directly

Reply:
EDIT: @tadfisher noted commenting on the type makes it show up as a doc comment which we dont want since the field is commented out